ingests_tracker/src/test/scala/uk/ac/wellcome/platform/storage/ingests_tracker/IngestsTrackerApiFeatureTest.scala

```diff
@@ -91,6 +94,43 @@ class IngestsTrackerApiFeatureTest
     }
   }
 
+  describe("PATCH /ingest/:id") {
+    val ingest = createIngestWith(
+      createdDate = Instant.now(),
+      events = Seq(createIngestEvent, createIngestEvent).sortBy {
+        _.createdDate
+      },
+      version = None
+    )
+
+    val ingestStatusUpdate =
+      createIngestStatusUpdateWith(
+        id = ingest.id,
+        status = Succeeded
+      )
+
+    // Cases:
+    // - Updating existing ingest 200
+    // - Updating existing ingest with bad data 500
+    // - Updating non-existent ingest 500
+    // - Broken app 500
```

Comment:
If you look at the IngestUpdater state machine logic, it's possible for it to reject an ingest update (e.g. if a previously Failed ingest gets a status update "Succeeded"). Maybe that's a 400, not a 500?

Reply:
I think probably 409. I'm going to do:

```
    // - Updating existing ingest with bad json 400
    // - Updating existing ingest to incorrect state 409
```